provision/acc_provision/acc_provision.py

```diff
@@ -1329,7 +1329,7 @@ def generate_cko_calico_yaml(config, network_operator_output):
             fh.write(network_operator_yaml)
 
 
-def generate_cko_aci_yaml(config, network_operator_output):
+def generate_cko_aci_yaml(args, config, network_operator_output):
```

Comment:
args -> version

and just assign that to netopConfig["aci_config"]["cni_flavor_version"], no conditional processing is required